### DISCUSSION TITLE
PS2 and N64 require separate board?

### DISCUSSION THREAD

**Discussion by Koohiisan** [darthcloud/BlueRetro#704] [General]
I am wanting to make a single BlueRetro that will work for my GC, N64, and PS2 via 25pin connectors.  It looks like my GC gets wired up directly to the ESP32, and I tested it as such and it works fine.  But the other two systems' wiring directions show a separate board being used if I read correctly.  I notice a few systems seem to show this, actually.

Is there information on that board that I overlooked?  Are they purchasable with all components soldered in already?

**Comment by GrechTech** [comment]
Hi
For PS2, GC and N64, you are fine with direct connections using cut cables.

Related to what you're looking for, [here ](https://www.tindie.com/products/grechtech/retrorosetta-universal-blueretro/) we sell $10 DIY kits to convert standard ESP32 DevKits into fully featured and cased (+$5) BlueRetro universal units. We also have a line of PCBs that connect to the boards with all active circuitry required for each system, to allow instantly swapping between systems, this is because many systems require some combination of pull-up resistors, level shifter ICs, power regulation or external power inputs, however as previously mentioned, most modern systems including PS2, GC and N64, do not require this: 
https://www.tindie.com/products/grechtech/retrorosetta-cables/

**Reply by Koohiisan** [reply]
Cool, thanks for the info!  I'll check those out!
